examples/lfric/eg17/README.md

```diff
@@ -0,0 +1,54 @@
+# PSyData Examples
+
+**Author:** J. Henrichs, Bureau of Meteorology
+
+This directory contains some runnable examples of LFRic. They are based
+on a subset of the LFRic infrastructure library, that has been slightly
+modified to make it easier to create stand-alone, non-MPI LFRic codes.
+The examples here show:
+- A [full_example](./full_example) of a stand-alone LFRic-based code. It
+  uses PSyclone for a very simple kernel.
+- [Extraction](./full_example_extract) of input- and output-parameter
+  of a kernel to NetCDF files.
+- A stand-alone example using [NetCDF](./full_example_netcdf) to read
+  in a mesh.
+
+Detailed instructions are in the various subdirectories.
```

Comment:
What does "various" mean? I suggest explicitly listing these subdirectories here or adding that information to the relevant bullet points above and deleting this sentence.

Reply:
I've slightly reworded this, hope it is better. The directories are listed just before that, each even with a link.